extensions/Khronos/KHR_mesh_compression/README.md

```diff
@@ -0,0 +1,172 @@
+# KHR_draco_geometry_compression
```

Comment:
We talked to Patrick and Neil and the plan was to preferably propose it as KHR extension and integrate it to glTF 2.0 if we can work something out before the 2.0 release soon. But we can start with GOOGLE and see how it goes it that's better?

The draco library could also compress point cloud, in that case the primitive will just not have indice accessor. Calling it geometry will be better since it includes mesh and point cloud.